src/screens/AddCash/providers/Coinbase/index.tsx

```diff
@@ -38,7 +38,7 @@ export function Coinbase({ accountAddress, config }: { accountAddress: string; c
             provider: FiatProviderName.Coinbase,
           });
 
-          Linking.openURL(url);
+          openInBrowser(url);
```

Comment:
@BrodyHughes Should just be these screens in `AddCash/` that we need to revert; lmk if you think anything else would be impacted by missing cookies